### ISSUE TITLE
Timit_asr dataset cannot be previewed recently

### ISSUE THREAD

**Issue by YingLi001** [huggingface/datasets#4169]
## Dataset viewer issue for '*timit_asr*'

**Link:** *https://huggingface.co/datasets/timit_asr*

Issue: The timit-asr dataset cannot be previewed recently.

Am I the one who added this dataset ? Yes-No
No

**Comment by severo** [comment]
Thanks for reporting. The bug has already been detected, and we hope to fix it soon.

**Comment by lhoestq** [comment]
TIMIT is now a dataset that requires manual download, see #4145 

Therefore it might take a bit more time to fix it

**Comment by YingLi001** [comment]
> TIMIT is now a dataset that requires manual download, see #4145
> 
> Therefore it might take a bit more time to fix it

Thank you for your quickly response. Exactly, I also found the manual download issue in the morning. But when I used *list_datasets()* to check the available datasets, *'timit_asr'* is still in the list. So I am a little bit confused. If *'timit_asr'* need to be manually downloaded, does that mean we can **not** automatically downloaded it **any more** in the future?

**Comment by lhoestq** [comment]
Yes exactly. If you try to load the dataset it will ask you to download it manually first, and to pass the downloaded and extracted data like `load_dataset("timir_asr", data_dir="path/to/extracted/data")`

The URL we were using was coming from a host that doesn't have the permission to redistribute the data, and the dataset owners (LDC) notified us about it.

**Comment by Nhamnguyen258** [comment]
I downloaded the timit_asr data and unzipped. But I can't run my code. Could you resolve this problem for me? Thanks
<img width="694" alt="image" src="https://user-images.githubusercontent.com/107462958/216513748-84e29ff2-5d78-40b8-a6b4-8a63b900409b.png">

<img width="799" alt="image" src="https://user-images.githubusercontent.com/107462958/216513799-31e3437c-2948-4ba3-8640-97c164522534.png">
